src/services/lookup-table-handler.ts

```diff
@@ -71,24 +71,24 @@ function createExtractor(protoLookupTable: DataTable, tableMatcher?: LookupTable
     };
   } else {
     logger.debug(`Using lookup table to try try to generate the extractor...`);
-    const sortColumn = protoLookupTable.dataTableDescriptions.find((info) =>
-      info.columnName.toLowerCase().startsWith('sort')
+    const sortColumn = protoLookupTable.dataTableDescriptions.find(
+      (info) => info.columnName.toLowerCase().indexOf('sort') > -1
```

Comment:
```suggestion
      (info) => info.columnName.toLowerCase().includes('sort')
```

Not a blocker but `includes()` is more readable IMO.